src/MatrixAlgebraKit.jl

```diff
@@ -30,6 +30,9 @@ export LAPACK_HouseholderQR, LAPACK_HouseholderLQ,
        LAPACK_DivideAndConquer, LAPACK_Jacobi
 export truncrank, trunctol, truncabove, TruncationKeepSorted, TruncationKeepFiltered
 
+VERSION >= v"1.11.0-DEV.469" &&
+    eval(Meta.parse("public default_algorithm, select_algorithm"))
```

Comment:
Alternative that does not require `parse` is simply
`eval(Expr(:public, :default_algorithm, :select_algorithm))`
but I am fine with `Meta.parse` (the overhead is probably negligible).

Reply:
Makes sense, I changed it to that.